core/storage/classifier/gae_models.py

```diff
@@ -175,13 +183,35 @@ def create(
             interaction_id=interaction_id,
             exp_id=exp_id,
             exp_version=exp_version,
-            state_name=state_name, status=status, training_data=training_data
+            next_scheduled_check_time=next_scheduled_check_time,
+            state_name=state_name, status=status,
+            training_data=training_data,
             )
 
         training_job_instance.put()
         return instance_id
 
     @classmethod
+    def query_training_jobs(cls, cursor=None):
+        """Queries the next ClassifierTrainingModel to be picked up VM.
```

Comment:
This is unclear. Perhaps:

     Gets the next 10 jobs which are either in status "new" or "pending", ordered by their next_scheduled_check_time attribute.